payment/controllers.go

```diff
@@ -581,3 +590,42 @@ func MerchantTransactions(service *Service) handlers.AppHandler {
 		return nil
 	})
 }
+
+// VerifyCredentialsRequest includes an order id and opaque subscription credential blob
+type VerifyCredentialsRequest struct {
+	OrderID     uuid.UUID                  `json:"orderId" valid:"-"`
```

Comment:
We don't want to include the order id when verifying credentials since this breaks the anonymity of the blinded tokens. We should probably include the SKU so we can lookup the issuer key instead?

Reply:
Sounds good @evq , will rm the orderID here. Had a question about subscriptions, In these kind of credential redemptions, what kind of payload should we pass to challenge bypass? 

the payload field here 
```
SELECT * FROM redemptions;
 id | issuer_type | ts | payload 
```